diff --git a/glib/gthread-posix.c b/glib/gthread-posix.c
index be960d3..81a8367 100644
--- a/glib/gthread-posix.c
+++ b/glib/gthread-posix.c
@@ -1207,14 +1207,14 @@ g_system_thread_new (GThreadFunc proxy,
 #endif /* _SC_THREAD_STACK_MIN */
       /* No error check here, because some systems can't do it and
        * we simply don't want threads to fail because of that. */
-      pthread_attr_setstacksize (&attr, stack_size);
+      //pthread_attr_setstacksize (&attr, stack_size);
     }
 #endif /* HAVE_PTHREAD_ATTR_SETSTACKSIZE */
 
 #ifdef HAVE_PTHREAD_ATTR_SETINHERITSCHED
     {
       /* While this is the default, better be explicit about it */
-      pthread_attr_setinheritsched (&attr, PTHREAD_INHERIT_SCHED);
+      //pthread_attr_setinheritsched (&attr, PTHREAD_INHERIT_SCHED);
     }
 #endif /* HAVE_PTHREAD_ATTR_SETINHERITSCHED */
 
