diff --git a/demos/gtk-demo/meson.build b/demos/gtk-demo/meson.build
index bd32cb57f2..96296be437 100644
--- a/demos/gtk-demo/meson.build
+++ b/demos/gtk-demo/meson.build
@@ -240,6 +240,7 @@ executable('gtk4-demo',
   dependencies: gtkdemo_deps,
   include_directories: confinc,
   win_subsystem: 'windows',
+  android_usecase: 'application',
   link_args: extra_demo_ldflags,
   install: true,
 )
diff --git a/gdk/android/glue/meson.build b/gdk/android/glue/meson.build
new file mode 100644
index 0000000000..f134d04a55
--- /dev/null
+++ b/gdk/android/glue/meson.build
@@ -0,0 +1,10 @@
+gdk_android_glue = static_library('glue', 'glue.c',
+  dependencies: [
+    glib_dep,
+    libgtk_dep,
+    cc.find_library('log')
+  ]
+)
+
+gdk_android_glue_dep = declare_dependency(link_whole: gdk_android_glue)
+meson.override_dependency('android:application', gdk_android_glue_dep)
diff --git a/meson.build b/meson.build
index b9f6190831..20c7529bcf 100644
--- a/meson.build
+++ b/meson.build
@@ -830,6 +830,9 @@ subdir('gdk')
 subdir('gsk')
 subdir('gtk')
 subdir('modules')
+if os_android
+  subdir('gdk/android/glue')
+endif
 if get_option('build-demos')
   subdir('demos')
 endif

